react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.swift

```diff
@@ -42,6 +43,15 @@ extension RNReactNativeGutenbergBridge {
     public override static func requiresMainQueueSetup() -> Bool {
         return true
     }
+
+    public override func batchDidComplete() {
+        if isJSLoading {
+            isJSLoading = false
+            DispatchQueue.main.async {
+                self.delegate?.gutenbergDidLoad()
```

Comment:
We are not using `gutenbergDidLoad` anymore, but I thought on keeping it for a while in case is needed in the future.